core/src/main/java/com/hcl/ecomm/core/services/impl/MySqlServiceImpl.java

```diff
@@ -0,0 +1,107 @@
+package com.hcl.ecomm.core.services.impl;
```

Comment:
@rakeshkumar-verma : pl add the runomdoe config too for default database connection.

Reply:
@rishabh-behl  mysql jdbc config already present in dev branch